webrtc/videoroom_advanced/lib/videoroom/room.ex

```diff
@@ -14,6 +14,13 @@ defmodule Videoroom.Room do
     GenServer.start(__MODULE__, init_arg, opts)
   end
 
+  @spec start_link([
+          {:debug, [:log | :statistics | :trace | {any, any}]}
+          | {:hibernate_after, :infinity | non_neg_integer}
+          | {:name, atom | {:global, any} | {:via, atom, any}}
+          | {:spawn_opt, [:link | :monitor | {any, any}]}
+          | {:timeout, :infinity | non_neg_integer}
+        ]) :: :ignore | {:error, any} | {:ok, pid}
```

Comment:
I am not sure if we want this typespec. I would remove it or use https://hexdocs.pm/elixir/1.13.1/GenServer.html#t:option/0 and  https://hexdocs.pm/elixir/1.13.1/GenServer.html#t:on_start/0